src/include/binder/bind_node_visitor.h

```diff
@@ -114,6 +117,32 @@ class BindNodeVisitor : public SqlNodeVisitor {
                                common::ErrorCode::ERRCODE_FEATURE_NOT_SUPPORTED);
     }
   }
+
+  std::vector<common::ManagedPointer<parser::AbstractExpression>> UnifyOrderByExpression(
```

Comment:
We are trying to improve our compilation times. Please move this function into `bind_node_visitor.cpp`

Reply:
I've modified this.